diagnostic.md

```diff
@@ -23,7 +26,7 @@ Write the jQuery code to navigate from one of the list item (`li`) DOM nodes to
 the `form` DOM node. (Assume the list item has a class of "listerItem").
 
 <!-- your answer starts here -->
-
+form.listerItem
```

Comment:
For this one we needed a jQuery selector again. Something like:

`$('li').parent().parent().find('form')`

This will start at the list item, go up to its parent, go up to that parent, and then look for a 'form' node.